### ISSUE TITLE
Connection limit exceeded

### ISSUE THREAD

**Issue by samuelcolvin** [rq/django-rq#76]
I'm getting an error running django-rq on Heroku:

```
redis.exceptions.ResponseError: max number of clients reached
```

The error is perfectly reasonably, I'm using the free redis cloud service which only allows 10 connections and running a script to generate content which calls django-rq lots of times and thereby exceeds the connection limit. But what's best fix?

I could probably force the queue to be synchronous for this task. But that only fixes this script, what happens in proper deployment if the system exceeds the `max number of clients` limit?

Basically is there an elegant way of using rq's connection pool in django-rq? Would it be easiest to use django-redis then specify `max_connections`?
